src/commands/git-metadata/gitdb.ts

```diff
@@ -99,7 +136,7 @@ export const uploadToGitDB = async (
       return
     }
     log.debug(`Uploading packfiles...`)
-    await uploadPackfiles(log, request, repoURL, latestCommits[0], packfiles)
+    await uploadPackfiles(log, request, repoURL, headCommit, packfiles)
```

Comment:
unrelated to this PR, but I've observed that in the `getObjectsToUpload` method ([code](https://github.com/DataDog/datadog-ci/blob/master/src/commands/git-metadata/gitdb.ts#L234-L243)), we're using the `commitsToExclude`. 

Would it be enough only use the `commitsToInclude`? Now we're doing:

```  
return {
    commitsToInclude: latestCommits.filter((x) => !commitsToExclude.includes(x)),
    commitsToExclude,
    headCommit: latestCommits[0],
  }
```

Should improve the performance of the command execution if we remove the `commitsToExclude`?

Reply:
but we need both, right? Do you mean to just use `latestCommits` and `commitsToExclude` and build `commitsToInclude` on the spot? That could be done but there would be no performance gains. Maybe it'd be clearer, but I'm not sure either